styleguide.config.js

```diff
@@ -27,7 +27,11 @@ module.exports = {
         },
         {
             name: "Dropdown",
+            content: "packages/wonder-blocks-dropdown/docs.md",
             components: [
+                "packages/wonder-blocks-dropdown/components/action-item.js",
+                "packages/wonder-blocks-dropdown/components/separator-item.js",
+                "packages/wonder-blocks-dropdown/components/option-item.js",
                 "packages/wonder-blocks-dropdown/components/action-menu.js",
                 "packages/wonder-blocks-dropdown/components/single-select.js",
                 "packages/wonder-blocks-dropdown/components/multi-select.js",
```

Comment:
Would be worth having an "Internal Components" section here with Check/Checkbox, as well? (I'm not sure if there are any other internal components but maybe they would be included, too.)

Reply:
I added the internal Dropdown component but not Check/Checkbox. The main reason I took those out was to clean up the `option-item.js` file, and I feel like they don't really serve as functional components that are worth pointing out because it's a bunch of styling that is very dependent on the OptionItem design anyways.